<?xml version="1.0"?>
<launch>
  <arg name="_output_interval" default="5"/>
  <node name="kf_evaluator" pkg="visualize" type="kf_evaluator.py" output="screen">
      <param name="_output_interval" value="$(arg _output_interval)"/>
  </node>
</launch>